### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by aubinleyoung**
## Vet clinic database: add "join table" for visits
In this milestone I:

- Created a table named vets
- Made the many-to-many relationship between the table
- Insert the given data for vets
- Wrote queries to answer the given questions
- Here you can find screenshots of the result of my queries 

Screenshot 2023-03-31 113306
Screenshot 2023-03-31 113437
Screenshot 2023-03-31 113824

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -36,4 +36,31 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
...
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
+    (1, 1, '2020-05-24'), -- Agumon visited William Tatcher on May 24th, 2020.
+    (1, 3, '2020-07-22'), -- Agumon visited Stephanie Mendez on Jul 22th, 2020.
+    (2, 4, '2021-02-02'), -- Gabumon visited Jack Harkness on Feb 2nd, 2021.
+    (3, 2, '2020-01-05'), (3, 2, '2020-03-08'), (3, 2, '2020-05-14'), -- Pikachu visited Maisy Smith on Jan 5th, Mar 8th, and May 14th, 2020.
+    (4, 3, '2021-05-04'), -- Devimon visited Stephanie Mendez on May 4th, 2021.
+    (5, 4, '2021-02-24'), -- Charmander visited Jack Harkness on Feb 24th, 2021.
+    (6, 2, '2019-12-21'), (6, 1, '2020-08-10'), (6, 2, '2021-04-07'), -- Plantmon visited Maisy Smith on Dec 21st, 2019, William Tatcher on Aug 10th, 2020, and Maisy Smith again on Apr 7th, 2021.
+    (7, 3, '2019-09-29'), -- Squirtle visited Stephanie Mendez on Sep 29th, 2019.
+    (8, 4, '2020-10-03'), (8, 4, '2020-11-04'), -- Angemon visited Jack Harkness on Oct 3rd and Nov 4th, 2020
```
Comment (AnselemOdims) [inline data.sql]: - [ ] Good job working on these, but please kindly include all missing insert data queries.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @aubinleyoung , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly ensure you include the screenshots of ALL queries in your PR description.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (aubinleyoung) [reply data.sql]: Thank you @AnselemOdims  for your comment 👍

**Review by codecaiine — APPROVED** [review]
Hi @aubinleyoung  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great Database setup ✔️ 
- Queries work as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**